en/docs/develop/customizations/customizing-the-developer-portal/styling-the-logo-and-header.md

```diff
@@ -1,9 +1,16 @@
 # Changing the Logo and Header Styles
 
 The header section can be customized to match different design needs. Following is the default look and the configuration.
+The default header of the API Manager Developer Portal is shown below.
 
  ![changing the logo and header]({{base_path}}/assets/img/learn/changing-the-logo-and-header1.png) 
 
+!!! note
+    -   You can find the default theme in the following location
+    `<APIM_HOME>/repository/deployment/server/jaggeryapps/devportal/source/src/defaultTheme.js`
+    -   Using the above file as reference you can override the parameters defined in the above file by altering the `defaultTheme.js` file in following location
```

Comment:
Shall we rephrase this as,
 
**Using the above file as a reference, you can override the parameters defined in the above file, by altering the `defaultTheme.js` file in the following location.**

Reply:
Fixed with 10584cd96c089244ce8914132e6b047ff59d5453

Reply:
This is fixed in the latest